### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by kbeker**
Update release notes to `v0.3.0` release

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 ### next
 
+### v0.3.0
```
Comment (Karrp) [inline RELEASE-NOTES.md]: There is no lack of information about this?
https://github.com/Code-Poets/sheetstorm/pull/123

Comment (kbeker) [reply RELEASE-NOTES.md]: You're right. Fixed

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 ### next
```
[the remark this answers is not in the document]
Comment (kbeker) [reply RELEASE-NOTES.md]: No, because it should be filling every pull request :)

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+- Bugfix 24 hours validation does not work on report creation ([#236](https://github.com/Code-Poets/sheetstorm/pull/236))
+- Bugfix add return button on user's report editting screen ([#181](https://github.com/Code-Poets/sheetstorm/pull/181))
+- Bugfix add styles to admin report detail ([#180](https://github.com/Code-Poets/sheetstorm/pull/180))
+- Bugfix change type annotations for get_reports_created ([#238](https://github.com/Code-Poets/sheetstorm/pull/238))
+- Bugfix changed sended pk while editing user in admin update template ([#176](https://github.com/Code-Poets/sheetstorm/pull/179))
+- Bugfix disable possibility to change report's author by Admin user ([#201](https://github.com/Code-Poets/sheetstorm/pull/201))
+- Bugfix for access to editing managers list and update tests ([#197](https://github.com/Code-Poets/sheetstorm/pull/197))
+- Bugfix show all project list ([#237](https://github.com/Code-Poets/sheetstorm/pull/237))
+- Bugfix wrong projects displayed after report creation fail ([#228](https://github.com/Code-Poets/sheetstorm/pull/228))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+##### Bugfixes
+
+- Bugfix 24 hours validation does not work on report creation ([#236](https://github.com/Code-Poets/sheetstorm/pull/236))
+- Bugfix add return button on user's report editting screen ([#181](https://github.com/Code-Poets/sheetstorm/pull/181))
+- Bugfix add styles to admin report detail ([#180](https://github.com/Code-Poets/sheetstorm/pull/180))
+- Bugfix change type annotations for get_reports_created ([#238](https://github.com/Code-Poets/sheetstorm/pull/238))
+- Bugfix changed sended pk while editing user in admin update template ([#176](https://github.com/Code-Poets/sheetstorm/pull/179))
+- Bugfix disable possibility to change report's author by Admin user ([#201](https://github.com/Code-Poets/sheetstorm/pull/201))
+- Bugfix for access to editing managers list and update tests ([#197](https://github.com/Code-Poets/sheetstorm/pull/197))
+- Bugfix show all project list ([#237](https://github.com/Code-Poets/sheetstorm/pull/237))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+- Feature rebuild user interface ([#123](https://github.com/Code-Poets/sheetstorm/pull/123))
+
+##### Bugfixes
+
+- Bugfix 24 hours validation does not work on report creation ([#236](https://github.com/Code-Poets/sheetstorm/pull/236))
+- Bugfix add return button on user's report editting screen ([#181](https://github.com/Code-Poets/sheetstorm/pull/181))
+- Bugfix add styles to admin report detail ([#180](https://github.com/Code-Poets/sheetstorm/pull/180))
+- Bugfix change type annotations for get_reports_created ([#238](https://github.com/Code-Poets/sheetstorm/pull/238))
+- Bugfix changed sended pk while editing user in admin update template ([#176](https://github.com/Code-Poets/sheetstorm/pull/179))
+- Bugfix disable possibility to change report's author by Admin user ([#201](https://github.com/Code-Poets/sheetstorm/pull/201))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,30 @@
...
+- Feature hours per month counter ([#140](https://github.com/Code-Poets/sheetstorm/pull/140))
+- Feature rebuild user interface ([#123](https://github.com/Code-Poets/sheetstorm/pull/123))
+
+##### Bugfixes
+
+- Bugfix 24 hours validation does not work on report creation ([#236](https://github.com/Code-Poets/sheetstorm/pull/236))
+- Bugfix add return button on user's report editting screen ([#181](https://github.com/Code-Poets/sheetstorm/pull/181))
+- Bugfix add styles to admin report detail ([#180](https://github.com/Code-Poets/sheetstorm/pull/180))
+- Bugfix change type annotations for get_reports_created ([#238](https://github.com/Code-Poets/sheetstorm/pull/238))
+- Bugfix changed sended pk while editing user in admin update template ([#176](https://github.com/Code-Poets/sheetstorm/pull/179))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,30 @@
...
+- Feature project report list username grouping ([#135](https://github.com/Code-Poets/sheetstorm/pull/135))
+- Feature project report list ([#133](https://github.com/Code-Poets/sheetstorm/pull/133))
+- Feature sum of hours of reports from a given day lower or euqal to 24 ([#207](https://github.com/Code-Poets/sheetstorm/pull/207))
+- Feature hours per day counter ([#137](https://github.com/Code-Poets/sheetstorm/pull/137))
+- Feature hours per month counter ([#140](https://github.com/Code-Poets/sheetstorm/pull/140))
+- Feature rebuild user interface ([#123](https://github.com/Code-Poets/sheetstorm/pull/123))
+
+##### Bugfixes
+
+- Bugfix 24 hours validation does not work on report creation ([#236](https://github.com/Code-Poets/sheetstorm/pull/236))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,30 @@
...
+- Bugfix add return button on user's report editting screen ([#181](https://github.com/Code-Poets/sheetstorm/pull/181))
+- Bugfix add styles to admin report detail ([#180](https://github.com/Code-Poets/sheetstorm/pull/180))
+- Bugfix change type annotations for get_reports_created ([#238](https://github.com/Code-Poets/sheetstorm/pull/238))
+- Bugfix changed sended pk while editing user in admin update template ([#176](https://github.com/Code-Poets/sheetstorm/pull/179))
+- Bugfix disable possibility to change report's author by Admin user ([#201](https://github.com/Code-Poets/sheetstorm/pull/201))
+- Bugfix for access to editing managers list and update tests ([#197](https://github.com/Code-Poets/sheetstorm/pull/197))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,30 @@
...
 
+### v0.3.0
+
+##### Features
+
+- Feature adjust templates for printing ([#136](https://github.com/Code-Poets/sheetstorm/pull/136))
+- Feature project report detail ([#134](https://github.com/Code-Poets/sheetstorm/pull/134))
+- Feature project report list username grouping ([#135](https://github.com/Code-Poets/sheetstorm/pull/135))
+- Feature project report list ([#133](https://github.com/Code-Poets/sheetstorm/pull/133))
+- Feature sum of hours of reports from a given day lower or euqal to 24 ([#207](https://github.com/Code-Poets/sheetstorm/pull/207))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,30 @@
...
+- Feature hours per day counter ([#137](https://github.com/Code-Poets/sheetstorm/pull/137))
+- Feature hours per month counter ([#140](https://github.com/Code-Poets/sheetstorm/pull/140))
+- Feature rebuild user interface ([#123](https://github.com/Code-Poets/sheetstorm/pull/123))
+
+##### Bugfixes
+
+- Bugfix 24 hours validation does not work on report creation ([#236](https://github.com/Code-Poets/sheetstorm/pull/236))
+- Bugfix add return button on user's report editting screen ([#181](https://github.com/Code-Poets/sheetstorm/pull/181))
```
Comment (maciejSamerdak) [inline RELEASE-NOTES.md]: 👍

**Pull request merged**